packages/forms/examples/forms/pages/search-filter/search-filter.page.ts

```diff
@@ -1,7 +1,60 @@
 import { Component } from '@angular/core';
+import { SearchFilterChoice } from '@acpaas-ui/ngx-components/forms';
 
 @Component({
 	templateUrl: './search-filter.page.html',
 })
 export class FormsSearchFilterDemoPageComponent {
+
+	public stuff: SearchFilterChoice[] = [{
+		label: "First item",
+		value: "one",
+	}, {
+		label: "Second item",
+		value: "two",
+	}, {
+		label: "Third item",
+		value: "three",
+	}, {
+		label: "Fourth item",
+		value: "four",
+	}];
+
+	public searchfilterImportExample = `import { SearchFilterModule } from '@acpaas-ui/ngx-components/forms';
+
+@NgModule({
+	imports: [
+		SearchFilterModule,
+	]
+});
+
+export class AppModule {};`;
+
+	public searchfilterExampleJS1 = `public stuff: SearchFilterChoice[] = [{
```

Comment:
You can add `import { SearchFilterChoice } from '@acpaas-ui/ngx-components/forms';` here.